gruntfile.js

```diff
@@ -119,8 +122,7 @@ module.exports = function (grunt) {
     dist: {
       src: [
         'public/javascripts/browsered.js',
-        'common/assets/javascripts/base/*.js',
-        'common/assets/javascripts/modules/*.js'
+        'common/assets/javascripts/**/*.js'
```

Comment:
The selective copying of `base/` and `modules/` folder looked like a hangover from when govuk code was copied into the assets. This new selector will concat any JavaScript assets on to the end of the browserified file.

Other code changes above ensure that the JS files are otherwise ignored by asset copying and watching.